functions-runtime/Makefile

```diff
@@ -30,7 +30,7 @@ release-image.arm64v8: clean
 	docker build  --load --no-cache --build-arg ARCH="arm64v8" -t $(RELEASE_REGISTRY)/$(RELEASE_IMAGE)-arm64 .
 
 .PHONY: push-release-images
-push-release-images: release-image.amd64 release-image.arm64v8
```

Comment:
you also removed amd64

Reply:
yes. it's redundant to build here, since this is just the push step